_components/form-controls/07-range-input.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Range input
```

Comment:
I think I prefer `range slider` to `range input`, but I don't feel too strongly about it. But be consistent through out the page

Reply:
It was range slider originally but changed it to input bc of a duplicate id issue: the ID of the input was `range-slider` as was the ID of the section heading. We could change it to slider but would have to make a change to the ID on the USWDS repo for this component.

Reply:
It seems like `range-slider` is the appropriate id for the component. Is there any way to use `Range slider` as the section name in the docs but give it a custom id?

Reply:
Added a custom control to change the section id in a9e0888 and updated the name to slider in 8546358.